app/controllers/users/verify_personal_key_controller.rb

```diff
@@ -26,7 +26,12 @@ def create
       else
         result = personal_key_form.submit
 
-        analytics.track_event(Analytics::PERSONAL_KEY_REACTIVATION_SUBMITTED, result.to_h)
+        analytics_result = FormResponse.new(
+          success: result.success?,
+          errors: result.errors,
+          extra: result.extra.except(:decrypted_pii),
```

Comment:
This would always be empty (as of today), so we could just omit `extra` altogether.